presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -253,6 +258,7 @@ private Environment getEnvironment()
                 container
                         // the test jar is hundreds MB and file system bind is much more efficient
                         .withFileSystemBind(testJar.getPath(), "/docker/test.jar", READ_ONLY)
+                        .withFileSystemBind(cliJar.getPath(), "/docker/presto-cli-executable.jar", READ_ONLY)
```

Comment:
we can hide the fact CLI jar is a jar:

```
.withFileSystemBind(cliBin.getPath(), "/docker/presto-cli", READ_ONLY)
```

you will also want to make the file inside docker executable

Reply:
sure. this file is already executable